service/topicService.go

```diff
@@ -127,7 +127,7 @@ func (threadService *ThreadServiceImpl) GetThread(id string) (*model.Thread, int
 }
 
 func (threadService *ThreadServiceImpl) UpdateThreadPost(updatedPost model.Post) (*model.Post, int) {
-	thread, statusCode := threadService.GetThread(*updatedPost.ThreadId)
+	thread, statusCode := threadService.GetThread(*updatedPost.ThreadId, "1")
```

Comment:
Denne vil vel alltid hente de 20 første postene i en tråd, som gjør at vi ikke får oppdatert post nr 21 og utover?

I så fall; Ser i [nodebb kontrolleren](https://github.com/NodeBB/NodeBB/blob/master/src/controllers/topics.js) at det går an å legge på en post-index `index` på spørringen som gir tilbake post med index om den finnes og de rundt. Kan kreve post-index av frontend for update og delete og sende videre?

Eksempel:
`https://community.staging.fellesdatakatalog.digdir.no/api/topic/92/post/40`

Reply:
Det blir et problem for update og delete, ja. Men kan ikke bruke post_index, i hvert fall ikke direkte. Fra dokumentasjonen: [The API response is largely unaffected by this parameter, it is used client-side...](https://docs.nodebb.org/api/read/#tag/topics/paths/~1api~1topic~1{topic_id}~1{slug}/get)

Reply:
Hmm, når jeg testet funket det fint? Se det kallet under "eksempel"

Reply:
Aha, ja. Hjelper å faktisk følge dokumentasjonen i stedet for vill gjetting :+1: 
(Jeg hadde lagt til post_index som et requestParam...)

Reply:
23aa5ab